services/workflows-service/src/customer/customer.controller.external.ts

```diff
@@ -38,7 +43,19 @@ export class CustomerControllerExternal {
         customerStatus: true,
         projects: true,
       },
-    });
+    }) as Customer & { projects: { id: string }[] };
+
+    if (projectName =='demo'){
+      await this.createDemoMockData(customerCreateModel, createdCustomer);
+    }
+    return createdCustomer;
+  }
+
+  private async createDemoMockData(customerCreateModel: CustomerCreateDto, createdCustomer: & { projects: { id: string }[] }) {
```

Comment:
It will be great if we will have this in more isolated place (scripts with all other seeds make sense)